tests/run_integration_tests.bat

```diff
@@ -1,3 +0,0 @@
-@echo off
```

Comment:
Integration tests were meant to test the return code of the tank command, but this is increasingly not relevant with the phasing out of the tank command when using the webapp.